libs/utils/serialization.py

```diff
@@ -0,0 +1,151 @@
+from collections import namedtuple, Iterable, OrderedDict
+import numpy as np
+import json
+
+from energy_model import (ActiveState, EnergyModelNode, EnergyModelRoot,
```

Comment:
I think it's alright to go over 80 chars for the imports, so you could have that in a single line and ditch the parentheses.